Challenges/LINQ/LINQ.Challenge.Tests/HardTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using FluentAssertions;
+using LINQ.Challenge.Models;
+using LINQ.Challenge.Models.DTOs;
+
+namespace LINQ.Challenge.Tests;
+
+public class HardTests
```

Comment:
Would be good to have some tests that require multiple linq statements
E.g. List of people aged 25 or over, ordered alphabetically, returned as a PersonDto - Where().OrderBy().Select().ToArray()

Or a add up all the distinct numbers that are greater than 50
Where().Distinct().Sum()

Reply:
Added to Hard - Unique and return as DTO, Get an order list of unique first names of people over a specified age